### DISCUSSION TITLE
Playground is not running.

### DISCUSSION THREAD

**Discussion by giantgoldcat** [naverz/zepeto-studio-kor#540] [4. General]
Playground is not running.
It was executed a few days ago, but do you know why?
When I click playground, it seems to be loading for a while, but there is no response.

**Comment by Zepeto-TechEvangelist** [comment]
This is a community that posts information about ZEPETO world development.
Please contact Customer Center for this issue.
https://support.zepeto.me/hc/en-us/requests/new
